### ISSUE TITLE
How to visualise val loss graph?

### ISSUE THREAD

**Issue by dmumtaz** [TimothyHTimothy/FAST-VQA#30]
Similar to the training loss, can you please share how to visualize the validation loss V/S epoch graph used for checking the performance and whether the model is overfitting?

**Comment by teowu** [comment]
For validation, we provide curves for SRCC, PLCC, KRCC, and RMSE computation. They are metrics to know whether the model is overfitting.
As validation is only conducted **once an epoch**, we believe the metrics are enough for monitoring the training and loss curves are not so important.
You can also simply copy the related code from "finetune_epoch" to "inference_set" function to monitor them, with wandb apis.
